update-docs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
where should this live?  Is the top-level fine?

Reply:
I'd put it in `.github/scripts/` or `.github/workflows/scripts/`

Looks like GHA permits configuring the working directory accordingly too: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/adding-scripts-to-your-workflow